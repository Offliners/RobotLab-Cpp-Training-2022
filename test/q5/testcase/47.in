(15+18/13)=*(1-20)=-18+5=+16-3/4=*17/18-10+8*20-(17-1)=
