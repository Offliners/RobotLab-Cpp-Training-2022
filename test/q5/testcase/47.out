16
-304
-317
-301
-151
